src/sentry/sentry_metrics/configuration.py

```diff
@@ -9,6 +9,10 @@
 
 import sentry_sdk
 
+# The maximum length of a column that is indexed in postgres. It is important to keep this in
+# sync between the consumers and the models defined in src/sentry/sentry_metrics/models.py
+MAX_INDEXED_COLUMN_LENGTH = 200
```

Comment:
Not sure if this is the right place to put this constant. I needed someplace which can be shared by both the consumer and indexer model. But happy to move it somewhere else if needed.

Reply:
It looks good to me, I guess another place to consider would be somewhere in `postgres` directory because this is a limitation with our postgres setup? (But that directory is in `indexer`)

Reply:
Wouldn't want to put this in the postgres directory. The limit is something which is agnostic of what underlying technology gets used.

Reply:
Fair enough